<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>818</width>
    <height>420</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/logo.svg</normaloff>:/icons/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QWebView" name="webView">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="url">
       <url>
        <string>http://qt.nokia.com/</string>
       </url>
      </property>
      <property name="renderHints">
       <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>818</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>&amp;Navigation</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionStop"/>
    <addaction name="actionReload"/>
   </widget>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionA_propos"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAide"/>
  </widget>
  <widget class="QToolBar" name="navigationBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="actionStop"/>
   <addaction name="actionReload"/>
   <addaction name="actionAccueil"/>
   <addaction name="separator"/>
   <addaction name="actionRechercher"/>
   <addaction name="actionFavoris"/>
   <addaction name="actionHistorique"/>
   <addaction name="separator"/>
   <addaction name="actionImprimer"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="urlBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="loadingBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/previous.svg</normaloff>:/icons/previous.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Précédent</string>
   </property>
   <property name="toolTip">
    <string>Charge la page précédente dans l'historique</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/next.svg</normaloff>:/icons/next.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Suivant</string>
   </property>
   <property name="toolTip">
    <string>Charge la page suivante dans l'historique</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/stop.svg</normaloff>:/icons/stop.svg</iconset>
   </property>
   <property name="text">
    <string>Sto&amp;p</string>
   </property>
   <property name="toolTip">
    <string>Arrete le chargement de la page web</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/reload.svg</normaloff>:/icons/reload.svg</iconset>
   </property>
   <property name="text">
    <string>Actualiser</string>
   </property>
   <property name="toolTip">
    <string>Recharge la page web</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/quit.svg</normaloff>:/icons/quit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quitter</string>
   </property>
   <property name="toolTip">
    <string>Quitter le navigateur</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAccueil">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/home.svg</normaloff>:/icons/home.svg</iconset>
   </property>
   <property name="text">
    <string>Page d'accueil</string>
   </property>
  </action>
  <action name="actionRechercher">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/find.svg</normaloff>:/icons/find.svg</iconset>
   </property>
   <property name="text">
    <string>Rechercher</string>
   </property>
  </action>
  <action name="actionFavoris">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/bookmarks.svg</normaloff>:/icons/bookmarks.svg</iconset>
   </property>
   <property name="text">
    <string>Favoris</string>
   </property>
  </action>
  <action name="actionHistorique">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/history.svg</normaloff>:/icons/history.svg</iconset>
   </property>
   <property name="text">
    <string>Historique</string>
   </property>
  </action>
  <action name="actionImprimer">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/print.svg</normaloff>:/icons/print.svg</iconset>
   </property>
   <property name="text">
    <string>Imprimer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPrevious</sender>
   <signal>triggered()</signal>
   <receiver>webView</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNext</sender>
   <signal>triggered()</signal>
   <receiver>webView</receiver>
   <slot>forward()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>webView</sender>
   <signal>statusBarMessage(QString)</signal>
   <receiver>statusBar</receiver>
   <slot>message(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>720</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReload</sender>
   <signal>triggered()</signal>
   <receiver>webView</receiver>
   <slot>reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionStop</sender>
   <signal>triggered()</signal>
   <receiver>webView</receiver>
   <slot>stop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
